src/shared/constants/regions.ts

```diff
@@ -0,0 +1,60 @@
+export const convertProviderSymbol = (sym: string) => symbolProviderMap[sym]
+
+const symbolProviderMap = {
+  zuora: 'Zuora Z-Commerce Platform',
+  aws: 'Amazon Web Services',
+  gcm: 'Google Cloud Platform',
+  azure: 'Microsoft Azure',
+}
+
+export const REGIONS = [
```

Comment:
I would reach out to deployments since this looks like an incomplete list of regions that's missing asia pacific, but might also be missing some others?

Reply:
had offline convo. This list was an older/hardcoded solution, and not following the recent quartz-as-src-of-truth approach. Undoing this consolidation of regions code.